### PR TITLE
Update changelogs for Containership 3.11 release

### LINKED ISSUE #38: Add plugins change log for all updated plugins
#### What needs to be completed

Plugins Change-log updates for all new plugins (and missing existing):
https://github.com/containership/plugins-changelog

* [x] Google CSI 0.2.0-alpha
* [x] Google CSI 0.4.0
* [x] Google CSI 0.5.1
* [x] AWS CSI 0.3.0
* [x] DO CSI 1.0.1
* [x] DO CSI 1.1.0
* [x] Containership Cluster Manager 5.2.2
* [x] Containership Cluster Manager 5.3.1
* ~Containership Cluster Manager 6.0.0~
* [x] Containership Cluster Manager 6.0.1
* [x] Calico CNI 3.3.1
* [x] Canal CNI 3.3.1
* [x] DO CCM 0.1.7
* [x] DO CCM 0.1.15
* [x] Autoscaler Cerebral 0.3.2-alpha
* [x] Autoscaler Cerebral 0.5.0-alpha
* [x] GPU Device 1.0.0-beta

### PR DESCRIPTION
**Pull request by mattkelly**
Obviously, we have a lot of style and consistency issues here. For now, I just tried to be consistent within each individual changelog.

Closes #38

### CODE REVIEW

File: gpu_device/nvidia/CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,22 @@
 
 ## Versions
 
+- [v1.0.0](#v100-beta)
```
Comment (nicktate) [inline gpu_device/nvidia/CHANGELOG.md]: v1.0.0-beta

**Comment by mattkelly** [comment]
Nice catches @nicktate - updated.

**Comment by nicktate** [comment]
Nice work on this PR and getting all these up to date. :+1: I know this one was tedious but needed

**Comment by mattkelly** [comment]
Certainly a lot easier with the necessary updates listed in the ticket!

**Pull request merged**